source/agora/node/Validator.d

```diff
@@ -383,6 +383,12 @@ public class Validator : FullNode, API
 
     protected override void acceptHeader (const(BlockHeader) header) @safe
     {
+        // First we must validate the header
+        if (auto err = this.ledger.validateBlockSignature(header))
+        {
+            log.warn("acceptHeader: Recieved header is not valid: {}", err);
```

Comment:
Typo. Also make it a `trace` IMO, as it's taking external input, anyone could trigger it.